predelete/predelete.go

```diff
@@ -62,6 +67,33 @@ func PreDeleteOperatorSecret(clientset *kubernetes.Clientset) error {
 	return nil
 }
 
+func PreDeleteEnterprisePortalSecret(clientset *kubernetes.Clientset) error {
+	fmt.Println("Running pre delete hook")
+	secrets, err := clientset.CoreV1().Secrets(os.Getenv("TYK_POD_NAMESPACE")).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	found := false
+	for _, value := range secrets.Items {
+		if value.Name == os.Getenv("ENTERPRISE_PORTAL_SECRET_NAME") {
+			err = clientset.CoreV1().Secrets(os.Getenv("TYK_POD_NAMESPACE")).Delete(context.TODO(), value.Name, metav1.DeleteOptions{})
```

Comment:
Same comment as before
```suggestion
			err = clientset.CoreV1().Secrets(os.Getenv(TykPodNamespaceEnvVar)).Delete(context.TODO(), value.Name, metav1.DeleteOptions{})
```